src/gui/generalsettings.cpp

```diff
@@ -54,17 +54,6 @@ GeneralSettings::GeneralSettings(QWidget *parent)
     _ui->autostartCheckBox->setChecked(Utility::hasLaunchOnStartup(Theme::instance()->appName()));
     connect(_ui->autostartCheckBox, &QAbstractButton::toggled, this, &GeneralSettings::slotToggleLaunchOnStartup);
 
-    // setup about section
-    QString about = Theme::instance()->about();
-    if (about.isEmpty()) {
```

Comment:
The branding can specify an empty about, in which case we should also not show the entry in the system tray.

Reply:
Fixed